src/routes/safe/components/Transactions/helpers/utils.ts

```diff
@@ -16,3 +18,12 @@ export const areSafeParamsEnabled = (parametersStatus: ParametersStatus): boolea
 export const ethereumTxParametersTitle = (isExecution: boolean): string => {
   return `Owner transaction ${isExecution ? '(Execution)' : '(On-chain approval)'}`
 }
+
+export const getInteractionTitle = (value: number | string): string => {
+  if (value === 0 || (typeof value === 'string' && parseFloat(value) === 0)) {
```

Comment:
`if (value === 0 || value === '0')` maybe?

Reply:
What about if it's something like '0.0000'?

Reply:
Then `if (Number(value) === 0)`. Should work for both strings and numbers.